mos-platform/rp6502/clock.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _CLOCK_H
+#define _CLOCK_H
+
+struct timespec {
+  long tv_sec;
```

Comment:
This should be `time_t tv_sec`.

Also, I'll probably get flak for this, but 2038 is only 14 years away, and I do expect this project to still be around by then. Given that, and how rarely time manipulation will probably end up being a performance bottleneck, it really seems like `time_t` should be `long long`.

Notably, if you don't feel the same about rp6502, then it would probably be fine to upcast from its internal 32-bit time to a 64-bit `time_t`.

Reply:
The RP6502 OS uses a long long internally, but cc65 doesn't support long long. I can make new OS calls that support long long (but not now). There's no hurry, the current ones promote fine, and it can be done in a non-breaking way at any time.

I populated common time.h with a long long time.h and removed clock.h.

I also populated rp6502.h with temporary stdio things. Feel free to delete these as needed.